### DISCUSSION TITLE
绳匠安全防范预警

### DISCUSSION THREAD

**Discussion by SoraGinko114** [share121/inter-knot#553] [其他]
各位绳匠：
由于市政换届选举临近，治安局加强了对非官方空洞探索活动的打击力度，并已开通专门热线，鼓励检举绳匠。
绳匠是绳网的重要用户，也是新艾利都的珍贵财富，绳网团队特别提醒各位注意活动安全，保护个人信息。愿您度过一段平安愉快的节日时光！
![5e9d6f40883120e0bc077713da1d043e400787120](https://github.com/user-attachments/assets/7969de25-1c69-42a3-8949-099cc12d94ae)

**Comment by BOXANK** [comment]
是吗，治安官不可能找到我的YA

**Comment by 1122246** [comment]
治安局来这儿发公告？给我干哪儿来了，这还是绳网吗？？？

**Comment by ZhuYuan-148** [comment]
治安局会积极打击违法犯罪行为。我们不鼓励各位绳匠忽视生命安全，随意进出空洞！
